docs/DeveloperGuide.md

```diff
@@ -251,77 +270,377 @@ _{Explain here how the data archiving feature will be implemented}_
+|`*`     | OOC      | organised user                               | delete categories associated with contacts                                         | declutter my address book when the category is no longer relevant                                       |
+|`*`     | OOC      | intermediate user                            | filter contacts based on social media platform                                     | find out whose social media contacts I have not gotten and request it from them                         |
+|`*`     | OOC      | user                                         | track when I last queried my contact's information                                 | find out who I have not communicated with for an extended period of time                                |
+|`*`     | EOA      | expert user                                  | customise the information presented to me when the app is initialised              | view the social media contacts of my close friends quickly without keying in additional prompts         |
+|`*`     | EOA      | frequent user                                | create keyboard shortcuts/hotkeys                                                  | quickly pull up the social media handles of a contact-of-interest in the least keystrokes possible      |
+|`*`     | COC      | user                                         | add a profile picture for my contacts                                              | better recognize their appearance                                                                       |
+|`*`     | COC      | user                                         | add notes about contacts                                                           | recall specific items about my contacts                                                                 |
+|`*`     | COC      | user                                         | add dates of special occasions (birthdays/appointments) associated with my contact | view important information of my contacts                                                               |
+|`*`     | COC      | user                                         | set reminders for special occasions associated with my contact                     | be alerted of these events                                                                              |
+|`*`     | COC      | user                                         | view a dashboard of significant events associated with my contact                  | be reminded of these dates                                                                              |
+|`*`     | COC      | user                                         | forward relevant details of an existing contact                                    | easily share such information upon request                                                              |
+
+
+
+
+### Use Cases <a name="use-cases"></a>
```

Comment:
I think this section might be a _bit_ messy, since there is a mixture of ordered list and unordered list! Maybe we can try just manually overriding the ordered list by using syntax such as `1\. blah blah blah` as indicated [here](https://meta.stackoverflow.com/questions/269657/numbered-list-with-1a-and-1b-using-markdown). Just my 2 cents!

Reply:
Thanks for the suggestion David! I agree that the current form seems jarring and cluttered as all of the user stories are presented in one gigantic table.  

I was actually thinking of dividing the current User Stories section into different sub-sections according to the various categories. For example:

**Core Functionalities**
Table of user stories in listed descending order of priority.

**Guide**
Table of user stories listed in descending order of priority.

Would you be amenable to this idea?

Reply:
I think thats a good idea, or alternatively can colour code the existing table's categories/priorities to make it more readable